examples/distributed_replicated_volume.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create an distributed replicated volume
+  hosts: gluster_servers
+  remote_user: root
+  gather_facts: false
+
+  vars:
+    gluster_cluster_hosts:
+      - 10.70.41.212
+      - 10.70.42.156
+    gluster_cluster_volume: testvol
+    gluster_cluster_replica_count: 3
+    gluster_cluster_force: 'yes'
+    gluster_cluster_transport: 'tcp'
+    gluster_cluster_bricks: '/mnt/brick1/b1,/mnt/brick2/b2,/mnt/brick2/b1'
+
+  roles:
+    - gluster.volume
```

Comment:
@devyanikota can you please move this to gluster-ansible-cluster, because that is where people are going to look for examples.